apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp

```diff
@@ -107,62 +107,76 @@ static void utc_media_MediaRecorder_setVolume_p(void)
 {
 	uint8_t prev, volume;
 	MediaRecorder mr;
-	unique_ptr<FileOutputDataSource> dataSource = unique_ptr<FileOutputDataSource>(new FileOutputDataSource(channels, sampleRate, pcmFormat, filePath));
 	mr.create();
-	mr.setDataSource(std::move(dataSource));
-	mr.prepare();
 	mr.getVolume(&prev);
-
-	TC_ASSERT_EQ_CLEANUP("utc_media_mediarecorder_setVolume", mr.setVolume(10), RECORDER_OK, goto cleanup);
-	mr.getVolume(&volume);
-	TC_ASSERT_EQ_CLEANUP("utc_media_mediarecorder_setVolume", volume, 10, goto cleanup);
+	if (mr.setVolume(prev) == RECORDER_ERROR_DEVICE_NOT_SUPPORTED) {
+		printf("device doesn not support adjust volume, it using default\n");
```

Comment:
meddbg

Reply:
this is tc, printf is correct.
Will apply another things.